README.md

```diff
@@ -1,5 +1,6 @@
 [![Build Status](https://travis-ci.org/gambol99/go-marathon.svg?branch=master)](https://travis-ci.org/gambol99/go-marathon)
 [![GoDoc](http://godoc.org/github.com/gambol99/go-marathon?status.png)](http://godoc.org/github.com/gambol99/go-marathon)
+[![Go Report Card](https://goreportcard.com/badge/github.com/katallaxie/go-marathon)](https://goreportcard.com/report/github.com/katallaxie/go-marathon)
```

Comment:
This points at the wrong repo. But really, I'd rather have this added by a separate PR as well.

Remember that if we have to revert the final commit that will end up in master for whatever functional reason, all drive-by changes will be gone as well.

Reply:
Lets keep it for now :)